src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -31,35 +31,38 @@
 			type="text"
 			:placeholder="t('spreed', 'Search participants')"
 			@input="handleInput">
-		<!-- Loading state -->
-		<Caption v-if="contactsLoading"
-			:title="t('spreed', 'Loading contacts')" />
-		<!-- List of possilbe participants -->
-		<Caption v-if="!contactsLoading"
-			:title="t('spreed', 'Select participants')" />
-		<ParticipantsList
+		<transition-group
+			v-if="hasSelectedParticipants"
+			name="zoom"
+			tag="div"
+			class="selected-participants">
+			<ContactSelectionBubble
+				v-for="participant in selectedParticipants"
+				:key="participant.label"
```

Comment:
key is not unique

Reply:
btw also id is not unique (have a user and a group named "admin")
So you need to create a custom key with source + id